messages/open.json

```diff
@@ -1,5 +1,5 @@
 {
-  "description": "open your default scratch org, or another org that you specify,\nTo open a specific page, specify the portion of the URL after \"yourInstance.salesforce.com/\" as --path.\nFor example, specify \"--path lightning\" to open Lightning Experience, or specify \"--path /apex/YourPage\" to open a Visualforce page.\nTo generate a URL but not launch your browser, specify --urlonly",
+  "description": "open your default scratch org, or another specified org\nTo open a specific page, specify the portion of the URL after \"yourInstance.salesforce.com/\" as --path.\nFor example, specify \"--path lightning\" to open Lightning Experience, or specify \"--path /apex/YourPage\" to open a Visualforce page.\nTo generate a URL but not launch it to your browser, specify --urlonly.",
```

Comment:
do we launch a url "**to** your browser?"  

alternatives:
launch it **in** your browser
launch it **via** your browser

Reply:
yep, you're right. I like "launch it IN your browser." i'll change.